docs/rfcs/rfc_0001_deductions_from_the_mission_statement.md

```diff
@@ -0,0 +1,111 @@
+# Deductions from the mission statement
+
+by: Crazy Techno Opossum
+
+This document is a Request For Comments. It is open for criticism and review.
+We expect review to improve the quality of this document.
+
+This document extracts some deductions and technical constraints from the mission statement.
+This document does not define a system that maps those constraints to a particular implementation.
+
+This document's intended audience is people that want to help build the system.
+Some cryptography and mathematical knowledge is expected.
+
+
+## Summary
+
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in
+[RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+
+
+## Key points from the mission statement
+
+> Imagine a unique social project that takes a web server and turns it into a multiplayer hub for 
+> a purposeful cause. The goal? To create a space where people come together, enjoy games or 
+> activities, and contribute to a shared fund. 
+> This fund isn’t just any fund – it’s a resource that automatically gets divided among the users of the server. 
+> So, the more you engage and participate, the more you receive from the fund. 
+> This innovative project not only brings people closer through fun and interaction but also 
+> enables them to collectively impact their community by distributing funds where they’re needed most.
+
+Technology wise, we extract the following points:
+- A web server.
+- Games and activities.
+- People can contribute to a shared fund.
+- The fund gets distributed among the users of the server.
+- A user's income from the fund is a function of their engagement.
+- Income is also distributed based on necessity.
```

Comment:
define necessity, we are not manually allocating funds, if there is necessity, users can intern share again, but we base level keep to the Linux design guideline of doing 1 function

Reply:
> define necessity
Intentionally not defined :grin: 

> we are not manually allocating funds, if there is necessity, users can intern share again, but we base level keep to the Linux design guideline of doing 1 function

The mission statement says: "distributing funds where they’re needed most".
We can take that sentence out if we're not considering necessity on fund distribution.
No strong opinions here.

Reply:
agreed :Dd 
lets take it out <3